src/routes/docs/+page.svelte

```diff
@@ -1 +1,30 @@
-<div class="mx-auto flex-grow justify-center px-4 sm:px-6 lg:px-8"></div>
+<script lang="ts">
+  import DocsBody from './components/DocsBody.svelte';
+  import Header from '../../components/Header/Header.svelte';
+  $: section = 1;
+  const brightBlue = '#5E69FF';
+  const cream = '#FEFAF3';
+
+  const logoMap = new Map([
+    [1, { logo: brightBlue, nav: 'navy' }],
+    [2, { logo: cream, nav: 'cream' }],
+    [3, { logo: brightBlue, nav: 'cream' }],
+    [4, { logo: brightBlue, nav: 'navy' }],
+    [5, { logo: cream, nav: 'cream' }],
+  ]);
+
+  // Section navigation titles for Aria
+
+  $: logoHex = logoMap.get(section)?.logo;
+  $: navColor = logoMap.get(section)?.nav;
+  // $: baseUrl = $page.url.protocol + '//' + $page.url.host;
+</script>
+
+<div class="min-h-[calc(100vh-100px)]">
+  <div class="relative lg:max-w-[100vw] xl:max-w-page">
+    <div class="absolute top-0 z-50 w-full"><Header {logoHex} {navColor} bind:section /></div>
+    <div class={`relative w-[100vw] min-w-full sm:pt-[80px] lg:pt-[140px] vertical-lg:w-[100%]`}>
+      <DocsBody />
+    </div>
+  </div>
+</div>
```

Comment:
I think you can get away with leaving `src/routes/docs/+layout.svelte` alone and just doing this:

```suggestion
<div>
  <DocsBody />
</div>
```

Then you can also remove all the other section, color, and such.

Reply:
I cannot get away with all, it then does not align with landing page, plus header is needed. I remove an outer class which had no effect

Reply:
You can reset `src/routes/docs/+layout.svelte` and have the header in there right?